services/autoscaling/src/simcore_service_autoscaling/api/health.py

```diff
@@ -21,7 +21,7 @@
 @router.get("/", include_in_schema=True, response_class=PlainTextResponse)
 async def health_check():
     # NOTE: sync url in docker/healthcheck.py with this entrypoint!
-    return f"{__name__}.health_check@{datetime.utcnow().isoformat()}"
+    return f"{__name__}.health_check@{datetime.datetime.now(datetime.timezone.utc).isoformat()}"
```

Comment:
I don't get it what's the difference here?

Reply:
read this [https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow)

![image](https://user-images.githubusercontent.com/35365065/214528702-49528614-dc3c-4e74-bcfd-28e379af3e2c.png)

Reply:
interesting. We should propagate this to the entire repo. We us the "utcnow" everywhere! Perhaps create an issue to not forgetting refactoring it

Reply:
Agree, I'd like to have a repo wide "timezone aware etc now"